src/init.rs

```diff
@@ -158,6 +158,12 @@ if [[ ${preexec_functions[(ie)starship_preexec]} -gt ${#preexec_functions} ]]; t
     preexec_functions+=(starship_preexec);
 fi;
 STARSHIP_START_TIME="$(date +%s)";
+function zle-keymap-select
+{
+    PROMPT=$(starship prompt --keymap=$KEYMAP);
```

Comment:
One last comment! (You actually pushed your commit right before I hit 'ENTER').

```suggestion
    PROMPT=$(starship prompt --keymap=$KEYMAP --jobs="$(jobs | wc -l)");
```

While there's no good way to keep the return status info while using character changing, we probably should keep the jobs info up.

Reply:
OK!

Reply:
I'll squash this changes to df8d5090bb8f718c7e92a8 too.